src/general/StageHUDBase.cs

```diff
@@ -283,9 +287,11 @@ public abstract class StageHUDBase<TStage> : Control, IStageHUD
     private bool compoundCompressed;
 
     /// <summary>
-    ///   This is opposite of the expected value, ie. this is true when the panels are collapsed
+    ///   These are the opposite of the expected value, ie. these are true when the panels are collapsed
```

Comment:
Referring to multiple elements in a summary is not recommended because the summary is documentation that is associated with a single syntax element. Either use a `<see cref="environmentPanelActive"/>` or a normal comment `// The following two variables are ...` here.